druid-shell/src/clipboard.rs

```diff
@@ -36,7 +36,7 @@ pub use crate::platform::clipboard as platform;
 /// data available as an SVG, for other editors, and a bitmap image for applications
 /// that can accept general image data.
 ///
-/// ## `FormatId`entifiers
+/// ## `FormatId` entifiers
```

Comment:
Maybe this one was intentional? because, like, "FormatIdentifiers". It isn't great, though

Reply:
oh :sweat_smile: missed it. 
Any suggestions? I would make it just `FormatId` :man_shrugging:

Reply:
This was intentional, but maybe not helpful. I'd either leave it as it was, or I would replace it with plain `FormatId`.

Reply:
leaving it